proto/cosmwasm/wasm/v1/authz.proto

```diff
@@ -85,14 +85,14 @@ message CombinedLimit {
 // message.
 // Since: wasmd 0.30
 message AllowAllMessagesFilter {
-  option (cosmos_proto.implements_interface) = "ContractAuthzFilterX";
+  option (cosmos_proto.implements_interface) = "cosmwasm.wasm.ContractAuthzFilterX";
```

Comment:
1. [registered interface here](https://github.com/cosmology-tech/wasmd/blob/5c39daee99b838482ec1bdd980bfa0a56213ec34/x/wasm/types/codec.go#L79)
2. [interface defined here](https://github.com/cosmology-tech/wasmd/blob/5c39daee99b838482ec1bdd980bfa0a56213ec34/x/wasm/types/authz.go#L216) 

would this make it `cosmwasm.wasm.ContractAuthzFilterX`?